include/cutlass/gemm/kernel/tile_scheduler_params.h

```diff
@@ -647,7 +647,7 @@ struct PersistentTileSchedulerSm90StreamKParams {
 
     // Calculate the number of stream-K units that would be needed if each stream-K unit
     // computed the minimum allowable k iterations. Truncate this to be in units of clusters.
-    auto cluster_size = cluster_shape.m() * cluster_shape.n();
+    uint64_t cluster_size = cluster_shape.m() * cluster_shape.n();
```

Comment:
I'm not sure how these three changes (that explicitly specify `uint64_t` instead of using `auto`) would fix warnings.

`cluster_shape` is a `GemmCoord`, so `m()` and `n()` should be `int`.  Thus, explicitly specifying `uint64_t` here should actually either introduce a warning (signed to unsigned conversion), or mask possible overflow in the product of two `int` values (by converting some negative value due to overflow to a large unsigned value).

Reply:
This part of the code changed since my original push, and I'm no longer getting a warning from the plain `auto`. I don't recall if I was getting an overflow warning to make it 64-bit... but in any event, I've reverted this.